src/components/TodoErrorList/TodoErrorList.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  setError: (error: boolean) => void,
+};
+
+export const TodoErrorList: React.FC<Props> = ({ setError }) => {
+  const [isErrorVisible, setIsErrorVisible] = useState(true);
+
+  const hidderError = () => {
```

Comment:
```suggestion
  const hideError = () => {
```
I guess using verb here looks a bit better